samples/nrf9160/fmfu_smp_svr/prj.conf

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable mcumgr.
+CONFIG_MCUMGR=y
+
+# Some command handlers require a large stack.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Enable the UART mcumgr transports.
+CONFIG_MCUMGR_SMP_UART=y
+CONFIG_UART_MCUMGR_RX_BUF_SIZE=512
+CONFIG_MCUMGR_BUF_SIZE=512
+
+# Required by the `taskstat` command.
+CONFIG_THREAD_MONITOR=y
+
+# Enable statistics and statistic names.
+CONFIG_STATS=y
+CONFIG_STATS_NAMES=y
+
+# Enable STAT and OS from the core commands
+CONFIG_MGMT_FMFU=y
+CONFIG_MCUMGR_CMD_OS_MGMT=y
+CONFIG_MCUMGR_CMD_STAT_MGMT=y
+
+CONFIG_NRF_MODEM_LIB=y
+CONFIG_NRF_MODEM_LIB_SYS_INIT=n
+CONFIG_NRF_MODEM_LIB_SHMEM_RX_SIZE=9000
+CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE=9000
```

Comment:
You should not need to override the defaults

Reply:
Oh okei, I'll remove those.